contrib/farmhash/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+
+add_library(farmhash ${CMAKE_CURRENT_SOURCE_DIR}/src/farmhash.cc)
+target_include_directories(farmhash SYSTEM PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/src/")
+set_target_properties(farmhash
+    PROPERTIES
+    LIBRARY_OUTPUT_DIRECTORY ${farmhash_BINARY_DIR}
```

Comment:
now the `framhash_BINARY_DIR` goes undefined, please update to CMAKE_CURRENT_BINARY_DIR

u should verify where the built library outputs